md/release-notes.md

```diff
@@ -83,6 +83,18 @@ Aside of the quality of life update of not having to run a migration between mig
 * A check has been added in Cluster mode as to forbid the start of nodes in different Bonita platform versions. For example, on a two-node cluster, you can't have a node in 7.11.0 and another in 7.11.1.
 The feature of having nodes in different Bonita versions was never supported, though there were cases where it "worked" in previous versions.  
 
+## Bonita Portal transformation
+
+Bonita Portal is being transformed into Bonita Applications. When Bonita Applications are ready, Bonita Portal will be removed.
+Developers and users will need to learn how to stop using the Portal and start using Bonita Applications instead.
+This change will allow Bonita and its users to get free from Google Web Toolkit (GWT) technology and offer opportunities for customization.
+Indeed, some Portal pages (with GWT inside) are being totally recreated with our own UI Designer. They will be customizable.
+Others (those that already were GWT-free) are being wrapped and will not be customizable.
```

Comment:
(those that already were already using another technology) instead of (those that already were GWT-free)

Reply:
fixed thanks